### ISSUE TITLE
Support for `symbol-macro` documentation issues

### ISSUE THREAD

**Issue by mariari** [melisgl/mgl-pax#18]
If one just loads their project in a typical way

```lisp
(asdf:load-system :project)
```

then one will get notes like

```lisp
WARNING:
   discarding unsupported DOCUMENTATION: doc-type MGL-PAX:SYMBOL-MACRO for object of type SYMBOL
WARNING:
   discarding unsupported DOCUMENTATION: doc-type MGL-PAX:SYMBOL-MACRO for object of type SYMBOL
WARNING:
   discarding unsupported DOCUMENTATION: doc-type MGL-PAX:SYMBOL-MACRO for object of type SYMBOL
WARNING:
   discarding unsupported DOCUMENTATION: doc-type MGL-PAX:SYMBOL-MACRO for object of type SYMBOL
```

At least on SBCL

However if one loads `mgl-pax` and then run the `load-system` it works fine.

```lisp
(defun load-docs-with-symbol-macro (&optional (ql nil))
  (if ql
      (progn (ql:quickload :mgl-pax/navigate)
             (ql:quickload :project-name))
      (progn (asdf:load-system :mgl-pax/navigate)
             (asdf:load-system :project-name))))
```

I find this odd, as the `defmethod` in the `mgl-pax` codebase should be in effect by the time it loads this piece of code.

Any idea what is causing this, or am I just loading the code incorrectly?

**Comment by melisgl** [comment]
Reproduced with:

    CL-USER> (asdf:load-system "mgl-pax")
    T
    CL-USER> (define-symbol-macro ggg nil)
    GGG
    CL-USER> (setf (documentation 'ggg 'pax:symbol-macro) "hello")
    WARNING:
       discarding unsupported DOCUMENTATION: doc-type MGL-PAX:SYMBOL-MACRO for object of type SYMBOL
    "hello"
    CL-USER> (asdf:load-system "mgl-pax/navigate")
    T
    CL-USER> (setf (documentation 'ggg 'pax:symbol-macro) "hello")
    "hello"

**Comment by melisgl** [comment]
Fixed in 37775bf. Thank you for the report.

**Comment by mariari** [comment]
Thank you for the quick response and fix
